plexapi/library.py

```diff
@@ -1093,20 +1116,21 @@ class Collections(PlexObject):
     def _loadData(self, data):
         self.ratingKey = utils.cast(int, data.attrib.get('ratingKey'))
         self._details_key = "/library/metadata/%s%s" % (self.ratingKey, self._include)
+        self.addedAt = utils.toDatetime(data.attrib.get('addedAt'))
+        self.childCount = utils.cast(int, data.attrib.get('childCount'))
+        self.collectionMode = data.attrib.get('collectionMode')
+        self.collectionSort = data.attrib.get('collectionSort')
```

Comment:
Maybe for a different PR:

For some collections, I don't see a `collectionMode` or `collectionSort` in my XML, even after changing the collection settings.

Reply:
Looks like `includePreferences=1` is needed in your url.